src/go/rpk/pkg/cli/cloud/cluster/secret/delete.go

```diff
@@ -0,0 +1,61 @@
+package secret
+
+import (
+	v1alpha2 "buf.build/gen/go/redpandadata/dataplane/protocolbuffers/go/redpanda/api/dataplane/v1alpha2"
```

Comment:
```suggestion
 "buf.build/gen/go/redpandadata/dataplane/protocolbuffers/go/redpanda/api/dataplane/v1alpha2"
```

Reply:
v1alpha2 is necessary by linter, I think the problem is the package name because is dataplanev1alpha2 and it should be v1alpha2